modules/ossm-rn-known-issues.adoc

```diff
@@ -31,6 +29,10 @@ While Kafka publisher is included in the release as part of Jaeger, it is not su
 
 These are the known issues in {ProductName}:
 
+* link:https://issues.redhat.com/browse/TRACING-1166[TRACING-1166] It is not currently possible to use the Jaeger streaming strategy within a disconnected environment. When a Kafka cluster is being provisioned, it results in a error: `Failed to pull image registry.redhat.io/amq7/amq-streams-kafka-24-rhel7@sha256:f9ceca004f1b7dccb3b82d9a8027961f9fe4104e0ed69752c0bdd8078b4a1076`.
+
+* link:https://issues.redhat.com/browse/TRACING-809[TRACING-809] Jaeger Ingester is incompatible with Kafka 2.3. When there are two or more instances of the Jaeger Ingester and enough traffic it will continuously generate rebalancing messages in the logs.  This is due to a regression in Kafka 2.3 that was fixed in Kafka 2.3.1.  For more information, see https://github.com/jaegertracing/jaeger/issues/1819[Jaegertracing-1819].	* link:https://issues.redhat.com/browse/TRACING-809[TRACING-809] Jaeger Ingester is incompatible with Kafka 2.3. When there are two or more instances of the Jaeger Ingester and enough traffic it will continuously generate rebalancing messages in the logs.  This is due to a regression in Kafka 2.3 that was fixed in Kafka 2.3.1.  For more information, see https://github.com/jaegertracing/jaeger/issues/1819[Jaegertracing-1819].
```

Comment:
Copy/paste error?  TRACING-809 is in there twice.  Or was it already documented and I just didn't see it before and then you added it again?